db/schema.rb

```diff
@@ -12,18 +12,18 @@
 
 ActiveRecord::Schema.define(version: 20170127000826) do
 
-  create_table "outcomes", force: :cascade do |t|
```

Comment:
This change is not pertinent to the feature you are implementing and probably doesnt need to be included in this commit